Source/BugsnagCrashReport.h

```diff
@@ -60,7 +60,8 @@ NSString *_Nonnull BSGFormatSeverity(BSGSeverity severity);
                               errorMessage:(NSString *_Nonnull)message
                              configuration:(BugsnagConfiguration *_Nonnull)config
                                   metaData:(NSDictionary *_Nonnull)metaData
-                                  severity:(BSGSeverity)severity;
+                                  severity:(BSGSeverity)severity
+                                 unhandled:(BOOL)unhandled;
```

Comment:
Since this interface is public, we may want to add a second constructor to avoid making this a breaking change.

Reply:
I've reverted the signature change here, as the extra parameter wasn't being used within the constructor, and any client calling this method will be passing a handled error.